examples/reliable_conversation/mcp_agent.config.yaml

```
@@ -0,0 +1,43 @@
+  default_model: "gpt-4"
+
+anthropic:
+  default_model: "claude-3-sonnet-20240229"
+
+rcm:
+  # Quality control settings
+  quality_threshold: 0.8
+  max_refinement_attempts: 3
+  consolidation_interval: 3
+  evaluator_model_provider: "openai" # or anthropic
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)